src/main/kotlin/frc/robot/lib/extensions/ChassisSpeedsExtensions.kt

```diff
@@ -0,0 +1,8 @@
+package frc.robot.lib.extensions
+
+import edu.wpi.first.math.kinematics.ChassisSpeeds
+import kotlin.math.hypot
+
+object ChassisSpeedsExtensions {
+fun ChassisSpeeds.getSpeed() = hypot(vxMetersPerSecond, vyMetersPerSecond)
+}
```

Comment:
This can be a top-level function, also you don't need multiple files for extensions., there are only a few of those